mypyc/primitives/list_ops.py

```diff
@@ -105,6 +105,14 @@
     c_function_name='CPyList_Count',
     error_kind=ERR_MAGIC)
 
+# list.insert(index, obj)
+method_op(
+    name='insert',
+    arg_types=[list_rprimitive, int_rprimitive, object_rprimitive],
+    return_type=int_rprimitive,
```

Comment:
the return type should be `c_int_rprimitive` since it corresponds to the low-level int.

Reply:
Ah, that makes sense. Should be ok now.